lib/src/layer/polygon_layer/polygon_layer.dart

```diff
@@ -165,9 +222,9 @@ class _PolygonLayerState extends State<PolygonLayer> {
             tolerance: tolerance,
             highQuality: true,
           ),
-          holePoints: holes == null
-              ? null
-              : List<List<DoublePoint>>.generate(
+          holePoints: holes.isEmpty
+              ? []
+              : List.generate(
```

Comment:
Probably redundant, List.generate(length: 0) shouldn't be a expensive operation.

Reply:
Resolved in 9475c7a.